1750208314619	Sync.LogManager	DEBUG	Flushing file log
1750208314621	Sync.LogManager	DEBUG	Log cleanup threshold time: 1749344314621
1750208314621	Sync.LogManager	DEBUG	Done deleting files.
1750211914833	Sync.Service	DEBUG	User-Agent: Zen/1.12.10b (Linux x86_64) FxSync/1.141.0.20250601014821.desktop
1750211914833	Sync.Service	INFO	Starting sync at 2025-06-17 21:58:34 in browser session xBw1mX2qbEsv
1750211914877	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1750211914878	Sync.Status	INFO	Resetting Status.
1750211914878	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1750211914882	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1750211914882	FirefoxAccounts	DEBUG	getOAuthToken enter
1750211914882	FirefoxAccounts	TRACE	getCachedToken returning cached token
1750211914882	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1750211914882	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1750211914883	Sync.SyncAuthManager	INFO	Getting sync key
1750211914883	FirefoxAccounts	DEBUG	getOAuthToken enter
1750211914883	FirefoxAccounts	TRACE	getCachedToken returning cached token
1750211914883	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1750211914883	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1750211914883	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1750211914883	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1750211914883	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1750211914888	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1750211914888	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]"}
1750211914892	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1750211914892	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1750211914892	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1750211914892	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1750211914893	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/open-uri
1750211914893	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1750211914893	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1750211914893	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/close-uri/v1
1750211914896	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/270368697/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1750211914896	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1750211914896	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1750211914898	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1750211914898	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1750211914898	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1750211914899	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
